bot.py

```diff
@@ -351,6 +353,31 @@ def __init__(self, *args, choices=None, opts_to_inv=None, **kwargs):
         )
         self.add_item(select_menu)
 
+class EmojiSyncView(discord.ui.View):
+    # Might be able to delete guild reference
+    def __init__(self, emoji, mod_type: str, old_name=None, *args, **kwargs):
+        super().__init__(timeout=None, *args, **kwargs)
+        self.emoji = emoji
+        self.old_name = old_name
+        
+        # Mod type will be a String of 'Add', 'Del', or 'Name' 
+        self.mod_type = mod_type
+
+    @discord.ui.button(label='Accept', style=discord.ButtonStyle.green)
+    async def accept_callback(self, button, interaction: discord.Interaction):
+        await interaction.response.send_message('Okay! I will sync this now', ephemeral=True)
+        
+        if self.mod_type == 'Add':
+            await sync_add(bot=bot, emoji=self.emoji)
+        elif self.mod_type == 'Del':
+            await sync_delete(bot=bot, emoji=self.emoji)
```

Comment:
When deleting an emoji in the hub server, this error was thrown:

/PittBOT/bot.py:1919: RuntimeWarning: coroutine 'sync_delete' was never awaited
  sync_delete(bot=bot, emoji=emoji)
RuntimeWarning: Enable tracemalloc to get the object allocation traceback

Reply:
Yep. Just a simple syntax error on my part. Found the error and just awaited it